docs/src/solvers.md

```diff
@@ -1,7 +1,84 @@
 Interacting with solvers
 ========================
 
-TODO: Describe the connection between JuMP and solvers. Automatic vs. Manual
-mode. CachingOptimizer. How to set/change solvers. How to set parameters (solver
+A JuMP model stores a
+[MathOptInterface (MOI)](https://github.com/JuliaOpt/MathOptInterface.jl)
+backend internally that contains the optimization solver. The JuMP layer on top
```

Comment:
"JuMP can be viewed as a lightweight user-friendly layer on top of the MOI backend.